### ISSUE TITLE
Request: make taskbar accessible to screenreader users.

### ISSUE THREAD

**Issue by willbilec** [farmerbb/Taskbar#305]
Hi,
I would really like to use this app. I use a screen reader. The way it works is it reads anything I swipe to or slide my finger over. If you want to experience it, go to Settings > Accessibility and turn on Talkback. While the main app is fully accessible, the actual taskbar is not. All apps simply have unlabeled buttons, and the app names should be dynamically spoken.
An example of a properly labeled button is the Start button on the far left.
To learn more about Android accessibility, you can visit:
https://developer.android.com/guide/topics/ui/accessibility
This app may also help:
https://support.google.com/accessibility/android/answer/6376570?hl=en
If you have any questions, please let me know.
Thanks!
